timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -514,12 +520,70 @@ public void leaderCanProcessMultiClientGetCommitTimestampRequest() {
+        List<NamespacedLeaderTime> leaderTimeResponses =
+                multiClientConjureTimelockService.leaderTimes(AUTH_HEADER, expectedNamespaces);
+        Set<String> namespacesWithLeaderTime = leaderTimeResponses.stream()
+                .map(NamespacedLeaderTime::getNamespace)
+                .collect(Collectors.toSet());
+        assertThat(namespacesWithLeaderTime).hasSameElementsAs(expectedNamespaces);
+
+        // Whether we hit the multi client endpoint or conjureTimelockService endpoint(services one client in one
+        // call), for a namespace, the underlying service to process the request is the same
+        leaderTimeResponses.forEach(namespacedLeaderTime -> {
+            LeaderTime conjureTimelockServiceLeaderTime = leader.client(namespacedLeaderTime.getNamespace())
+                    .namespacedConjureTimelockService()
+                    .leaderTime();
+            assertThat(namespacedLeaderTime.getLeaderTime().id()).isEqualTo(conjureTimelockServiceLeaderTime.id());
+        });
+    }
+
+    @Test
+    public void sanityCheckMultiClientGetCommitTimestampsAgainstConjureTimelockService() {
+        TestableTimelockServer leader = cluster.currentLeaderFor(client.namespace());
+        MultiClientConjureTimelockService service = leader.multiClientService();
+        Set<String> expectedNamespaces = ImmutableSet.of("alpha", "beta");
+        List<NamespacedGetCommitTimestampsResponse> commitTimestamps = service.getCommitTimestamps(
+                AUTH_HEADER, defaultNamespacedGetCommitTimestampsRequests(expectedNamespaces));
+        Set<String> namespaces = commitTimestamps.stream()
+                .map(NamespacedGetCommitTimestampsResponse::getNamespace)
+                .collect(Collectors.toSet());
+        assertThat(namespaces).hasSameElementsAs(expectedNamespaces);
+
+        // Whether we hit the multi client endpoint or conjureTimelockService endpoint(services one client in one
+        // call), for a namespace, the underlying service to process the request is the same
+        commitTimestamps.forEach(namespacedGetCommitTimestampsResponse -> {
+            GetCommitTimestampsResponse conjureTimelockServiceGetCommitTimestampResponse = leader.client(
+                            namespacedGetCommitTimestampsResponse.getNamespace())
+                    .namespacedConjureTimelockService()
+                    .getCommitTimestamps(defaultCommitTimestampRequest());
+            assertThat(namespacedGetCommitTimestampsResponse
+                            .getLockWatchUpdate()
+                            .logId())
+                    .isEqualTo(conjureTimelockServiceGetCommitTimestampResponse
+                            .getLockWatchUpdate()
+                            .logId());
+            assertThat(namespacedGetCommitTimestampsResponse.getInclusiveUpper())
+                    .isEqualTo(conjureTimelockServiceGetCommitTimestampResponse.getInclusiveLower() - 1);
```

Comment:
nit/suggestion: Usually these assertions are `assertThat(actual).isEqualTo(expected)`, so I'd probably flip these two. Also, you can use `as(String)` to say in English what the assertion is doing, which I think is useful for this later one, which verifies `timestamps are contiguously increasing per namespace if there are no elections`.